internal/logger/logger.go

```diff
@@ -0,0 +1,99 @@
+package logger
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"go.uber.org/zap"
+)
+
+var (
+	setLog string
+	envLog = os.Getenv("OPCAP_LOG_LEVEL")
+	cfg    zap.Config
+)
+//LogJSON implements the structure of Zap Logger
+type LogJSON struct {
+	EncoderConfig *EncoderCfg `json:"encoderConfig"`
+	Level         string      `json:"level"`
+	Encoding      string      `json:"encoding"`
+	OutputPaths   []string    `json:"outputPaths"`
+}
+//EncoderCfg implements the structure for EncoderConfig field.
+type EncoderCfg struct {
+	MessageKey   string `json:"messageKey"`
+	LevelKey     string `json:"levelKey"`
+	LevelEncoder string `json:"levelEncoder"`
+}
+
+var logger *zap.Logger
+var sugarLogger *zap.SugaredLogger
+//InitLogger creates the Sugared Zap Logger
+//logLever could be set either through the flag "--log-level" or environment variable OPCAP_LOG_LEVEL
+//which take precedence respectively. 
+func InitLogger(logLevel string) {
+	setLog = envLog
+	if logLevel != envLog {
+		setLog = logLevel
+	}
+	data := &LogJSON{
+		Level:       setLog,
+		Encoding:    "json",
+		OutputPaths: []string{"stdout"},
+		EncoderConfig: &EncoderCfg{
+			MessageKey:   "message",
+			LevelKey:     "level",
+			LevelEncoder: "lowercase",
+		},
+	}
+	dataJSON, err := json.Marshal(data)
+	if err != nil {
+		log.Fatalf("failed to build zap logger: %v", err)
+	}
+
+	if err := json.Unmarshal([]byte(dataJSON), &cfg); err != nil {
+		panic(err)
+	}
+	logger, err = cfg.Build()
+	if err != nil {
+		panic(err)
+	}
+	sugarLogger = logger.Sugar()
+}
+//Info exports Info Suggared Loglevel.
+func Info(message string, fields ...interface{}) {
+	sugarLogger.Info(message, fields)
+}
+//Debug exports Suggared Loglevel.
```

Comment:
A couple of small nits: 

- start comment with lower case
- leave space before starting a comment
- remove period for consistency across all comment